src/libcrun/status.c

```diff
@@ -106,7 +106,7 @@ read_pid_stat (pid_t pid, struct pid_stat *st, libcrun_error_t *err)
 
   ret = xasprintf (&pid_stat_file, "/proc/%d/stat", pid);
   if (UNLIKELY (ret == -1))
-    return crun_make_error (err, errno, "xasprintf failed");
+    return crun_make_error (err, 0, "xasprintf failed");
```

Comment:
we could drop completely error check on `xasprintf` as it aborts on allocation failures.